src/Components/CompletedQuiz/CompletedQuiz.tsx

```diff
@@ -0,0 +1,86 @@
+import { Link } from "react-router-dom";
+
+export default function CompletedQuiz() {
```

Comment:
You can consider making the name of the component as plural to indicate that it holds all completed quizzes.